tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApiNetStd20.approved.txt

```diff
@@ -7,62 +7,69 @@
+        public static System.Collections.Generic.IList<T> CollectionOfDummy<[System.Runtime.CompilerServices.NullableAttribute(2)]  T>(int numberOfDummies) { }
         public static System.Collections.Generic.IList<T> CollectionOfFake<T>(int numberOfFakes)
             where T :  class { }
         public static System.Collections.Generic.IList<T> CollectionOfFake<T>(int numberOfFakes, System.Action<FakeItEasy.Creation.IFakeOptions<T>> optionsBuilder)
             where T :  class { }
-        public static T Dummy<T>() { }
+        public static T Dummy<[System.Runtime.CompilerServices.NullableAttribute(2)]  T>() { }
         public static T Fake<T>()
             where T :  class { }
         public static T Fake<T>(System.Action<FakeItEasy.Creation.IFakeOptions<T>> optionsBuilder)
             where T :  class { }
     }
-    public class static A<T>
+    [System.Runtime.CompilerServices.NullableAttribute(0)]
+    public class static A<[System.Runtime.CompilerServices.NullableAttribute(2)]  T>
     {
         public static T Ignored { get; }
         public static FakeItEasy.INegatableArgumentConstraintManager<T> That { get; }
         [System.CLSCompliantAttribute(false)]
         public static T _ { get; }
     }
+    [System.Runtime.CompilerServices.NullableAttribute(0)]
     public class static ArgumentConstraintManagerExtensions
     {
         public static string Contains(this FakeItEasy.IArgumentConstraintManager<string> manager, string value) { }
-        public static T Contains<T>(this FakeItEasy.IArgumentConstraintManager<T> manager, object value)
+        public static T Contains<[System.Runtime.CompilerServices.NullableAttribute(0)]  T>(this FakeItEasy.IArgumentConstraintManager<T> manager, object value)
             where T : System.Collections.IEnumerable { }
         public static string EndsWith(this FakeItEasy.IArgumentConstraintManager<string> manager, string value) { }
         public static System.Threading.CancellationToken IsCanceled(this FakeItEasy.IArgumentConstraintManager<System.Threading.CancellationToken> manager) { }
-        public static T IsEmpty<T>(this FakeItEasy.IArgumentConstraintManager<T> manager)
+        public static T IsEmpty<[System.Runtime.CompilerServices.NullableAttribute(0)]  T>(this FakeItEasy.IArgumentConstraintManager<T> manager)
             where T : System.Collections.IEnumerable { }
-        public static T IsEqualTo<T>(this FakeItEasy.IArgumentConstraintManager<T> manager, T value) { }
-        public static T IsGreaterThan<T>(this FakeItEasy.IArgumentConstraintManager<T> manager, T value)
+        public static T IsEqualTo<[System.Runtime.CompilerServices.NullableAttribute(2)]  T>(this FakeItEasy.IArgumentConstraintManager<T> manager, T value) { }

```

Comment:
I understand why this `T` is nullable here, but not why it's "oblivious" in `IsEmpty`…